osquery/tables/networking/posix/interfaces.cpp

```diff
@@ -93,6 +98,8 @@ void genDetailsFromAddr(const struct ifaddrs* addr, QueryData& results) {
   }
   r["mac"] = macAsString(addr);
 
+  size_t flags = addr->ifa_flags;
```

Comment:
Please do the same filtering here.

Reply:
These would need to fetch the flags for both linux and non linux. I dont think it should be filtered otherwise we would have missing flags for non linux.